Main/ClientApp/src/Components/Dialogs/EditContributorDialog.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState, ChangeEvent, useEffect } from "react";
+import SimpleDialog from "./SimpleDialog";
+import { updateSharedCollection } from "../../Actions/CollectionActions";
+import {
+  Button,
+  Avatar,
+  MenuItem,
+  FormControl,
+  InputLabel,
+  Select,
+  makeStyles,
+  Theme,
+  createStyles,
+  Grid,
+} from "@material-ui/core";
+import { Person } from "@material-ui/icons";
+import {
+  SharedCollectionData,
+  SharedCollection,
+} from "../../Model/SharedCollection";
+import SharedCollectionStore from "../../Stores/SharedCollectionsStore";
+import DeleteContributorDialog from "./DeleteContributorDialog";
+
+type EditContributorDialogProps = {
+  open: boolean;
+  toggleDialogOpen: () => void;
+  collectionId: number;
+  userId: number;
+};
+
+enum UserRights {
```

Comment:
Isn't this enum defined elsewhere? Move it to a seperate file and use it instead.

Reply:
i told you last time to put it in different file but you said to keep it as it is

Reply:
Because it was not used anywhere else so there was no reason to do so. Now there is, so it should be moved.